common/component_interface_tools/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.14)
+project(component_interface_tools)
```

Comment:
Do you plan to add other tools to this package?

Reply:
Yes. I would like to add a tool to replay service calls from `/service_log`.